bricks/template/README.md

```diff
@@ -14,6 +14,7 @@ A few resources to get you started if this is your first brick template:
 - [Official Mason Documentation][2]
 - [Code generation with Mason Blog][3]
 - [Very Good Livestream: Felix Angelov Demos Mason][4]
+- Dependency Injection (DI), State Management, and Navigating with [get_it](https://pub.dev/packages/get_it), [flutter_riverpod](https://pub.dev/packages/flutter_riverpod), and [go_router](https://pub.dev/packages/go_router).
```

Comment:
@doannimble This file instead and exact on that line https://github.com/nimblehq/flutter-templates/blob/ab5559ccd8c79dd819f1f538303d4a4f54b92148/README.md?plain=1#L10, please 🙏